cool-core/src/main/java/com/nus/cool/core/cohort/CohortUserSection.java

```diff
@@ -103,23 +103,22 @@ public boolean isCohortsInCublet() {
     }
 
     @Override
-    public void process(ChunkRS chunk) {
+    public void process(ChunkRS dataChunk) {
         totalDataChunks++;
 
         // process filters
-        sigma.process(chunk);
+        sigma.process(dataChunk);
         if (sigma.isUserActiveChunk() == false) {
             totalSkippedDataChunks++;
             return;
         }
 
-        FieldRS userField = chunk.getField(tableSchema.getUserKeyField());
+        FieldRS userField = dataChunk.getField(tableSchema.getUserKeyField());
         
         // Skipping non RLE compressed blocks
-        int totalCorruptedUsers = 0;
 
         if (!(userField.getValueVector() instanceof RLEInputVector)) {
-            totalCorruptedUsers++;
+            LOG.info("The user record corrupted: " + totalDataChunks);
```

Comment:
maybe just leave "user record corrupted". totalDataChunk does not give much info.

Reply:
@KimballCai